src/main/java/dan200/computercraft/shared/pocket/peripherals/PocketSpeaker.java

```diff
@@ -74,6 +75,7 @@ else if ( entity != null )
                 speaker.setLocation( entity.getEntityWorld(), entity.posX, entity.posY, entity.posZ );
             }
             speaker.update();
+            access.setLight( speaker.madeSound() ? 0x3320fc : -1 );
```

Comment:
`0x3320fc` is a random number just floating around in here, usually you'd add a comment or make a constant like `DEFAULT_LIGHT_COLOUR`, but I suppose it's already quite self-evident in this case.